packages/ra-ui-materialui/src/input/SelectArrayInput.spec.tsx

```diff
@@ -303,17 +304,131 @@ describe('<SelectArrayInput />', () => {
                 <Form
                     validateOnBlur
                     onSubmit={jest.fn()}
-                    render={() => (
-                        <SelectArrayInput
-                            {...{
-                                ...defaultProps,
-                            }}
-                        />
-                    )}
+                    render={() => <SelectArrayInput {...defaultProps} />}
                 />
             );
 
             expect(queryByRole('progressbar')).toBeNull();
         });
     });
+
+    test('should support creation of a new choice through the onCreate event', async () => {
```

Comment:
`test` used instead of `it` again... Please make sure there aren't any remaining in your PR

Reply:
We use both in the whole codebase btw